examples/architectureComponentsExample/src/main/java/io/realm/examples/arch/ExampleViewModel.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.examples.arch;
+
+import android.arch.lifecycle.ViewModel;
+
+import io.realm.Realm;
+import io.realm.examples.arch.livemodel.LiveRealmObject;
+import io.realm.examples.arch.model.Person;
+
+
+public class ExampleViewModel extends ViewModel {
```

Comment:
> For true offline support, you should always read data from the database, but that is orthogonal to using the repository pattern.

oh so **that's why I never got it**! We always had to support offline mode, or even when we didn't, I architected it in an "offline-first" way.

Not much point to `forceReload`ing a `RealmResults` though, it's not like it can be out of date, heh.